torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -649,13 +673,13 @@ def inner_fn(*args):
             if trace_joint:
                 assert isinstance(args, tuple) and isinstance(args[0], (list, tuple))
                 tokens = args[0][:num_tokens]
+                assert all(token.numel() == 0 for token in tokens)
                 args = (args[0][num_tokens:], *args[1:])
             else:
                 tokens = args[:num_tokens]
+                assert all(token.numel() == 0 for token in tokens)
```

Comment:
? (why did we move + double the assert)

Reply:
I would have to declare `tokens=[]` outside, type checker was not happy,  giving lints about specifying type of that empty list :)